test/unit/rules/lint-no-access-key-test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+const message =
+  'No access key attribute allowed. Inconsistencies between keyboard shortcuts and keyboard comments used by screenreader and keyboard only users create a11y complications.';
```

Comment:
Message should be imported from rule file instead of duplicated.

Reply:
done